src/mono/System.Private.CoreLib/src/ILLink/ILLink.Descriptors.xml

```diff
@@ -618,6 +618,15 @@
 		<type fullname="Mono.ValueTuple`3" preserve="fields"/>
 		<type fullname="Mono.ValueTuple`4" preserve="fields"/>
 		<type fullname="Mono.ValueTuple`5" preserve="fields"/>
+		<!-- aot-compiler.c -->
+		<type fullname="Mono.I8Enum" preserve="fields"/>
```

Comment:
Should we be splitting these off into a separate exclusions file when building the AOT compiler? Is that an option?

Reply:
These types need to be in the linked corlib which is AOTed. They are just small types with no members.